sdk/spring/azure-spring-boot/README.md

```diff
@@ -60,6 +60,8 @@ The following section provides sample projects illustrating how to use the Azure
 ### More sample code
 - [Azure Active Directory for WebApi](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server)
 - [Azure Active Directory for WebApp](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp)
+- [Azure Active Directory for Resource Server](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-spring-security-resource-server)
+- [Azure Active Directory for WebApi Obo](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo)
```

Comment:
`Azure Active Directory for WebApi` is same to `Azure Active Directory for WebApi Obo`.

Reply:
Changed to `Azure Active Directory for Resource Server with Obo Clients`.